tests/test_session.py

```diff
@@ -16,10 +18,39 @@ def newstylemodel(self):
             datetime_prop=datetime.datetime.now(datetime.timezone.utc)
         )
 
+    @pytest.fixture()
+    def raw_session_maker(self):
```

Comment:
this fixture is only used by the fixture `raw_session` maybe we don't need to separate it out?